include/ggml/ggml.h

```diff
@@ -2305,6 +2305,7 @@ extern "C" {
 
     GGML_API int          gguf_get_n_kv(const struct gguf_context * ctx);
     GGML_API int          gguf_find_key(const struct gguf_context * ctx, const char * key);
+    GGML_API int          gguf_find_key_array(const struct gguf_context * ctx, const char * key, const char * val);
```

Comment:
No need to add `gguf_find_key_array()` - it's result is never used for anything, so we can simply remove it